### ISSUE TITLE
Reduce number of tests

### ISSUE THREAD

**Issue by dkurt** [UNN-ITMM-Software/devtools-course-practice#85]
i.e. 256 test cases introduces in https://github.com/UNN-ITMM-Software/devtools-course-practice/pull/59 by `testing::Combine`.

@Alexandr-Solovev, did you mean only 4 tests by the following instantiation?
```cpp
INSTANTIATE_TEST_CASE_P(/**/, Solovev_Aleksandr_ComplexNumberTest_Parametrized,
                        testing::Combine(
  testing::Values(10.0, 3.0, 13.0, 3.0),
  testing::Values(0.0, 4.0, 0.0, 1.0),
  testing::Values(2.0, 0.0, 13.5, 0.0),
  testing::Values(0.0, 0.0, 0.0, 0.0)
));
```
Now it works in this way:
```
[----------] 256 tests from Solovev_Aleksandr_ComplexNumberTest_Parametrized
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/0, where GetParam() = (10, 0, 2, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/0 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/1, where GetParam() = (10, 0, 2, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/1 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/2, where GetParam() = (10, 0, 2, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/2 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/3, where GetParam() = (10, 0, 2, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/3 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/4, where GetParam() = (10, 0, 0, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/4 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/5, where GetParam() = (10, 0, 0, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/5 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/6, where GetParam() = (10, 0, 0, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/6 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/7, where GetParam() = (10, 0, 0, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/7 (0 ms)
11: [ RUN      ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/8, where GetParam() = (10, 0, 13.5, 0)
11: [       OK ] Solovev_Aleksandr_ComplexNumberTest_Parametrized.Test_Multiply/8 (0 ms)
...
```

**Comment by WerWebWer** [comment]
I see the same case in #76 pull requests from @BFDestroyeer 

```cpp
INSTANTIATE_TEST_CASE_P(/**/, Zoreev_Mikhail_ComplexNumberTest_Parametrized,
    testing::Combine(
  testing::Values(5.0, 3.0, 5.0, 3.0),
  testing::Values(0.0, 7.0, 4.0, 8.0),
  testing::Values(2.0, 6.0, 7.0, 1.0),
  testing::Values(3.5, 3.5, 3.5, 3.5)
));
```

**Comment by Alexandr-Solovev** [comment]
@dkurt, @WerWebWer,  fix it in https://github.com/UNN-ITMM-Software/devtools-course-practice/pull/94

**Comment by dkurt** [comment]
This issue is related to more tests. See 
[test_zoreev_mikhail_complex_number.cpp](https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/modules/complex-number/test/test_zoreev_mikhail_complex_number.cpp)
[test_rustamov_azer_complex_number.cpp](https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/modules/complex-number/test/test_rustamov_azer_complex_number.cpp)

**Comment by RustamovAzer** [comment]
Fixed my tests in #95.

**Comment by BFDestroyeer** [comment]
Fixed in #96
